README.md

```diff
@@ -1,10 +1,10 @@
-![](https://github.com/karam084/My_Portfolio_WebSite-)
+![]online version (https://karam084.github.io/portfolio_desktop_version/)
```

Comment:
- [ ] You have done a good job deploying your portfolio. But there is an issue with your page because it only shows the mobile version of it. I would suggest fixing this issue in order to show the desktop version for a better user experience :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/139769113-47b7f767-7c4a-4538-8490-5f1345666e19.png)

- [ ] I would suggest using the correct syntax in order to show the link for the live demo. You could use the [Microverse README template](https://github.com/microverseinc/readme-template) as a starting point. As you can see the "Live Demo Link" is in its own bullet/section :wink:

  ![image](https://user-images.githubusercontent.com/81522435/139769407-ff217bcc-3259-4aad-abe7-7c7122beeabe.png)